src/dom-builder.ts

```diff
@@ -243,13 +249,22 @@ function handleEntryClass(desc: string, elm: HTMLElement): void {
   });
 }
 
+const propKeywords = ["style", "attrs", "props", "class", "actionDefinitions", "actions", "setters", "entry", "behaviors", "streams", "output"];
```

Comment:
We could use a [`Set`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) here. It should give better performance.